.buildkite/pipeline.yml

```diff
@@ -11,6 +11,7 @@ common_params:
         repo: "automattic/woocommerce-ios/"
   # Common environment values to use with the `env` key.
   - &common_env
+    # Be sure to also update the `.xcode-version` file when updating the Xcode image/version here
     IMAGE_ID: xcode-14
```

Comment:
Great addition 👍 

In the past, I used [dynamic pipelines](https://github.com/buildkite/dynamic-pipeline-example), where instead of having an hardcoded YAML pipeline we'd run a script that generated the YAML at runtime. In that setup, we could read `.xcode-version` and generate the `IMAGE_ID` name based on it.

But, dynamic pipelines are harder to read, might have bugs in the generation logic, and we don't change Xcode versions _that_ fast to justify the investment and tradeoff at this point in time.